topology/topology_test.go

```diff
@@ -88,27 +89,41 @@ func myLog(c *check.C, args ...interface{}) {
 
 // TestHostMarshaling tests marshaling/unmarshaling of Host
 // structure to/from proper JSON.
+// Note: we don't actually do this anywhere. common.HostMessage
+// is used for that purpose
 func (s *MySuite) TestHostMarshaling(c *check.C) {
+	ip := net.ParseIP("10.1.1.1")
+	if ip == nil {
+		c.Error(fmt.Errorf("ParseIP failed"))
+	}
+	addr, nw, err := net.ParseCIDR("192.168.0.1/16")
+	if err != nil {
+		c.Error(err)
+	}
+	prefixLen, _ := nw.Mask.Size()
+
 	host := Host{}
 	host.Id = 1
-	host.RomanaIp = "192.168.0.1/16"
-	host.Ip = "10.1.1.1"
+	host.Ip = ip
+	host.RomanaIp = addr
+	host.RomanaPrefixLen = uint8(prefixLen)
 	host.Name = "host1"
 	host.AgentPort = 9999
 	m := common.ContentTypeMarshallers["application/json"]
 	json, _ := m.Marshal(host)
 	marshaledJSONStr := string(json)
 	myLog(c, "Marshaled ", host, "to", marshaledJSONStr)
-	expectedJSONStr := "{\"id\":1,\"name\":\"host1\",\"ip\":\"10.1.1.1\",\"romana_ip\":\"192.168.0.1/16\",\"agent_port\":9999}"
+	expectedJSONStr := `{"id":1,"name":"host1","ip":"AAAAAAAAAAAAAP//CgEBAQ==","romana_ip":"AAAAAAAAAAAAAP//wKgAAQ==","romana_net":16,"agent_port":9999}`
```

Comment:
I only updated this test because it exists. It isn't actually used to produce JSON for the REST API. (`common.HostMessage` does that, and data there is formatted as a string.)